README.md

```diff
@@ -31,16 +29,13 @@ HPP-FCL & Pinocchio must be built from sources. Build pinocchio with the flag :
 > [!NOTE]
 > Don't forget to switch to the right commits!
 
-## Usage
-Before trying the examples, test your hppfcl installation. To do this and make sure the hppfcl librairy works well on your computer, run in the test folder :
-``` python -m unittest```.
-
 ### Possible issue
-If you have a problem with ```FakeCollisionGeometry```, it is likely that the linking of Pinocchio with HPPFCL wasn't done properly. Verify that you have the right commits & the right compilation flags.
+- If you have a problem with ```FakeCollisionGeometry```, it is likely that the linking of Pinocchio with HPPFCL wasn't done properly. Verify that you have the right commits & the right compilation flags.
+- The main branch of HPPFCL doesn't compute well the closest points and thus, this repo needs to be built upon the devel branch. If it built but doesn't avoid collision, make sure that you didn't built the main branch.
 
-### For the MPC part:
+### For the OCP part:
 
-Create a meshcat-server by writting in a terminal ```meshcat-server```. Then, to see the example with hard constraints simply run in the main directory ```python examples/mpc_panda_reaching.py```. To compare it with the standard MPC without hard constraint but collision avoidance written as soft constraint, run  ```python examples/mpc_panda_reaching_soft_constraint.py```.
+Create a meshcat-server by writting in a terminal ```meshcat-server```. Then, to see the example with hard constraints simply run in the main directory ```python examples/main_ocp.py -s i```, where i is the index of the scenario, going from 1 to 3.
```

Comment:
```suggestion
Create a meshcat-server by writting in a terminal ```meshcat-server --zmq-url "tcp://127.0.0.1:6002"```. Then, to see the example with hard constraints simply run in the main directory ```python examples/main_ocp.py -s i```, where i is the index of the scenario, going from 1 to 3.
```
Starting `meshcat-server` externally not always guarantees that the port will be 6000, sometimes it starts at a different port and then needs to be manually adjusted. At least in my case when I wanted to start it with port 6000 I not always could do so

Reply:
let's have a meshcat server run every time the example is ran